docs/compute/_supported_methods_block_storage.rst

```diff
@@ -55,6 +55,7 @@ Provider                              list volumes create volume destroy volume
 `Opsource`_                           no           no            no             no            no            no             no             
 `Outscale INC`_                       yes          yes           yes            yes           yes           yes            yes            
 `Outscale SAS`_                       yes          yes           yes            yes           yes           yes            yes            
+`Packet`_                             no           no            no             no            no            no             no             
```

Comment:
Just for a future reference - in case you updated those files manually, they are generated automatically using `./contrib/generate_provider_feature_matrix_table.py` script.